go.mod

```diff
@@ -9,13 +9,12 @@ require (
 	github.com/gorilla/mux v1.6.2
 	github.com/jinzhu/copier v0.0.0-20180308034124-7e38e58719c3
 	github.com/lib/pq v1.0.0
+	github.com/modprox/mp v0.2.1 // indirect
```

Comment:
this module requires itself? we should figure out why this got added and fix it

Reply:
probably because of github.com vs oss.indeed.com, maybe it changed while I was working on this issue.